packit_service/config.py

```diff
@@ -224,6 +224,23 @@ def get_project_to_sync(self, dg_repo_name, dg_branch) -> Optional[ProjectToSync
 
 
 class PackageConfigGetter:
+    @staticmethod
+    def create_issue_if_needed(
+        project: GitProject, message: str, details: str = ""
+    ) -> Optional[Issue]:
+        issues = project.get_issue_list(author=project.service.user.get_username())
+
+        if "[packit] Invalid config" in (x.title for x in issues):
+            return None
+
+        if details:
+            details = f": {details}"
+
+        # TODO: store in DB
```

Comment:
Do you want to create just a model, or something more sophisticated?

Reply:
I am not sure about the benefits vs space complexity for this o.o
I guess @lbarcziova can provide better answer, since git blame blames her ;) I cannot seem to find any related issue, so I don't know what were the intentions :/

Reply:
If you look deeper, the TODO comment was added [here](https://github.com/packit/packit-service/commit/7361bf92e03f2d0a7ba62fc80025d337187d258b), so I am not the right person to answer..but I looked into that PR and this comment was probably added based on [this](https://github.com/packit/packit-service/pull/328#discussion_r366765240)

Reply:
thanks :)

Reply:
OK, it looks like Tomáš has wanted to store it to avoid the issue searching. And as Maťo said, we are filtering it by `packit` user, it's not such a big problem.